### PR TITLE
switch CI system to circleci

### PR DESCRIPTION
**Pull request by lopezm1**
Swapping CI system to circleci -- could I get a review? @rbm @rogerhu @asherf

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+aliases:
+  - &install-pip |
```
Comment (rogerhu) [inline .circleci/config.yml]: what is &?

Comment (rogerhu) [reply .circleci/config.yml]: never mind, found it -- https://circleci.com/blog/decrease-your-build-times-by-running-jobs-in-parallel-with-workflows/

**Comment by rogerhu** [comment]
is this tied to the open source plan or own plan?

**Comment by lopezm1** [comment]
@rogerhu Running under hearsaycorp organization but it's eligible for an open source setting. This means we get 3 free containers to run for this project.

**Pull request merged**